crispy_forms/tests/results/bootstrap4/test_layout/test_use_custom_control_is_used_in_checkboxes_false.html

```diff
@@ -1,8 +1,8 @@
 <form method="post">
-    <div class="form-group" id="div_id_checkboxes"><label class=" requiredField" for="">Checkboxes<span
```

Comment:
Could pull this change into a separate PR. Is it right that this is blank? 🤔

Reply:
I think this is fine. While Django puts the first inputs reference here the [bootstrap docs](https://getbootstrap.com/docs/4.6/components/forms/#horizontal-form) don't show this and accessibility tests don't complain either.

Wave complains about the layout but that's due to not using a fieldset and a legend. This is a separate issue.